pcap/pcap.go

```diff
@@ -103,7 +103,11 @@ int pcap_set_rfmon(pcap_t *p, int rfmon) {
 
 // The things we do to avoid pointers escaping to the heap...
 int pcap_next_ex_escaping(pcap_t *p, uintptr_t pkt_hdr, uintptr_t pkt_data) {
-  return pcap_next_ex(p, (struct pcap_pkthdr**)(pkt_hdr), (const u_char**)(pkt_data));
```

Comment:
Could you add a comment pointing to https://github.com/the-tcpdump-group/libpcap/blob/1131a7c26c6f4d4772e4a2beeaf7212f4dea74ac/pcap.c#L398-L406 and detailing the fact that it's possible this next_ex may return a value 1 or greater for success?

Reply:
Done in 0381271